webidl/publication_link.webidl

```diff
@@ -1,7 +1,7 @@
 dictionary PublicationLink {
-    required DOMString                  url;
-             DOMString                  encodingFormat;
-             DOMString                  name;
-             sequence<DOMString>        rel;
-             sequence<PublicationLink>  children;
```

Comment:
A UA will need to parse the HTML containing the ToC to extract it. I think that there's no reason to use a separate structure for handling the ToC when the WebIDL can play that role as well.